test/functional/cmdLineTests/shareClassTests/URLHelperTests/URLHelperTests_80.xml

```diff
@@ -43,7 +43,8 @@
 	<if testVariable="SCMODE" testValue="205" resultVariable="currentMode" resultValue="$mode205$"/>
 	<if testVariable="SCMODE" testValue="206" resultVariable="currentMode" resultValue="$mode206$"/>
 	
-		
+	<variable name="enableTimestampCheck" value="-Xshareclasses:name=URLHelperTests,checkClasspathTimestamp"/>
```

Comment:
Please move this up with the modes. Note that if this test is run in mode205, it won't work.

Reply:
Try specifying this as `,checkClasspathTimestamp` and then use it without a space after the current mode `$currentMode$$enableTimestampCheck$`

Reply:
Already updated the file as suggested and verified the test:
```
Testing: URLStaleClassPathEntryTest 4
Test start time: 2019/07/04 11:51:18 Pacific Standard Time
Running command: "/.../j2sdk-image/jre/bin/java" -Xcompressedrefs -Xcompressedrefs -Xjit -Xgcpolicy:gencon  
-Xshareclasses:name=URLHelperTests,checkClasspathTimestamp -Xbootclasspath/p:./Utils:.; StaleClassPathEntryTests.URLStaleClassPathEntryTest -testfile ./StaleClassPathEntryTests/props_unix/Test4.props -javacdir /root/jchau/temp/issues_2019/6022_sc_timestamp/j2sdk-image_v8/bin
Time spent starting: 1 milliseconds
Time spent executing: 5239 milliseconds
Test result: PASSED
```